.travis.yml

```diff
@@ -63,12 +80,6 @@ jobs:
 #      script:
 #        - go run main.go run dht/find-peers --builder=exec:go --runner=local:exec
 
-    - stage: run
```

Comment:
So no more smoke testing? This will expose us to refactor/reshuffle erros until we have wide test coverage.

Reply:
The smoke testing is covered here on Linux: https://github.com/ipfs/testground/pull/165/files#diff-354f30a63fb0907d4ad57269548329e3R32 No docker thing because that's flaky, and we're playing tricks with networks in a way that Travis likely won't like.